### PR TITLE
Sprints/finalcountdown

### PR DESCRIPTION
**Pull request by pierluc-ssoucy**

### CODE REVIEW

File: src/Solution/Solution/Entities/opc_informationcentre/RibbonDiff.xml [hunk 1]
```diff
@@ -23,6 +23,26 @@
...
+      </CommandUIDefinition>
+    </CustomAction>
+    <CustomAction Id="opc.Mscrm.HomepageGrid.opc_informationcentre.Flows.RefreshCommandBar.RunWorkflow.CustomAction" Location="Mscrm.HomepageGrid.opc_informationcentre.Flows.RefreshCommandBar.Flows.Controls._children" Sequence="30">
+      <CommandUIDefinition>
+        <FlyoutAnchor Alt="$Resources:RefreshCommandBar.RunWorkflow" Command="Mscrm.Form.Flows.RunWorkflow" Id="Mscrm.HomepageGrid.opc_informationcentre.Flows.RefreshCommandBar.RunWorkflow" Image16by16="/_imgs/Ribbon/OpenFlows_16.png" Image32by32="/_imgs/Ribbon/OpenFlows_32.png" LabelText="$Resources:RefreshCommandBar.RunWorkflow" PopulateDynamically="true" PopulateQueryCommand="Mscrm.DynamicMenu.Grid.Flows.PopulateWorkFlowMenu" Sequence="30" TemplateAlias="o1" ToolTipTitle="$Resources:Ribbon.HomepageGrid.Data.Workflow.RunWorkflow" ToolTipDescription="$Resources(EntityDisplayName):Ribbon.Tooltip.RunWorkflow" ModernImage="Flows" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_informationcentre/RibbonDiff.xml:43]: @pierluc-ssoucy all new code in this ribbon is not needed necessary anymore as the change is global. remove before merging.

File: src/Solution/Solution/PluginAssemblies/Plugins-CA0F1327-5D70-42E6-BA7C-8C4B5F342A5F/Plugins.dll.data.xml [hunk 2]
```diff
@@ -47,9 +47,6 @@
     <PluginType AssemblyQualifiedName="Plugins.PostLegalFileCreatePlugin, Plugins, Version=1.0.0.0, Culture=neutral, PublicKeyToken=da10840344fb2548" PluginTypeId="e80ce5b4-1d23-4b43-8d2e-30105506c067" Name="Plugins.PostLegalFileCreatePlugin">
       <FriendlyName>22a68335-2285-4aa6-8d0f-cc75394592db</FriendlyName>
     </PluginType>
-    <PluginType AssemblyQualifiedName="Plugins.ReassignRestrictedBusinessUnitPlugin, Plugins, Version=1.0.0.0, Culture=neutral, PublicKeyToken=da10840344fb2548" PluginTypeId="4de36ca0-238b-402e-bd79-f52ad2988f4c" Name="Plugins.ReassignRestrictedBusinessUnitPlugin">
```
Comment (GabrielFG) [inline src/Solution/Solution/PluginAssemblies/Plugins-CA0F1327-5D70-42E6-BA7C-8C4B5F342A5F/Plugins.dll.data.xml:50]: Normal that this has been removed?

File: src/Solution/Solution/Entities/opc_prpa_parlaffairsfile/FormXml/main/{12c80b09-fb79-46da-87a7-5674931fcadd}.xml [hunk 3]
```diff
@@ -272,107 +272,7 @@
...
-        <tab name="opc_tasks_tab" id="{002987e8-7cb2-b153-171d-72c0dd661765}" IsUserDefined="0" locklevel="0" showlabel="true" expanded="true">
-          <labels>
-            <label description="Tasks" languagecode="1033" />
-            <label description="Tâches" languagecode="1036" />
-          </labels>
-          <columns>
-            <column width="100%">
-              <sections>
-                <section name="opc_tasks_sectiion" showlabel="false" showbar="false" locklevel="0" id="{7b5639c0-bc87-b223-3ec8-286d80bcc92a}" IsUserDefined="0" layout="varwidth" columns="1" labelwidth="115" celllabelalignment="Left" celllabelposition="Left">
+                <section name="opc_tasks_sectiion" showlabel="true" showbar="false" locklevel="0" id="{7b5639c0-bc87-b223-3ec8-286d80bcc92a}" IsUserDefined="0" layout="varwidth" columns="1" labelwidth="115" celllabelalignment="Left" celllabelposition="Left">
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_prpa_parlaffairsfile/FormXml/main/{12c80b09-fb79-46da-87a7-5674931fcadd}.xml:275]: name typo

File: src/Solution/Solution/Entities/opc_fcproceduralstep/FormXml/main/{bcc7200b-b91f-43bc-bc63-2416ef5d6cc1}.xml [hunk 4]
```diff
@@ -173,7 +173,6 @@
         <tab id="{10fa06f9-5ff8-4c8b-8aee-4958d88a0154}" IsUserDefined="0" expanded="false">
           <labels>
             <label description="Notes" languagecode="1033" />
-            <label description="Remarques" languagecode="1036" />
           </labels>
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_fcproceduralstep/FormXml/main/{bcc7200b-b91f-43bc-bc63-2416ef5d6cc1}.xml:176]: add french back

File: src/Solution/Solution/Roles/Ci2 - Complaint Registrar.xml [hunk 5]
```diff
@@ -99,7 +99,7 @@
     <RolePrivilege name="prvCreateImportFile" level="Basic" />
     <RolePrivilege name="prvCreateImportMap" level="Basic" />
     <RolePrivilege name="prvCreateNewProcess" level="Global" />
-    <RolePrivilege name="prvCreateNote" level="Basic" />
+    <RolePrivilege name="prvCreateNote" level="Global" />
```
Comment (joshcampbell191) [inline src/Solution/Solution/Roles/Ci2 - Complaint Registrar.xml:102]: Is this intended?

File: src/Solution/Solution/Entities/opc_prpa_parlaffairsfile/FormXml/main/{12c80b09-fb79-46da-87a7-5674931fcadd}.xml (hunk 3 above)
Comment (pierluc-ssoucy) [reply src/Solution/Solution/Entities/opc_prpa_parlaffairsfile/FormXml/main/{12c80b09-fb79-46da-87a7-5674931fcadd}.xml:275]: was like that before, so we'll keep it this way.

File: azure-pipelines/scripts/update_connection_references.ps1 [hunk 6]
```diff
@@ -97,33 +98,24 @@ $solutionId = $solutions.CrmRecords[0].solutionid
...
-                if ($wf.clientdata.Contains($key)) {
-                    $impersonationCallerId = $connRefOwnerCollection[$key]
-                    break
-                }
-            }
-            if ($impersonationCallerId -ne "") {
-                $impersonationConn.OrganizationWebProxyClient.CallerId = $impersonationCallerId
-                Write-Host "Turning on $($wf.category) '$($wf.name)' with ID $($c.objectid)."
-                Set-CrmRecordState -conn $impersonationConn -EntityLogicalName workflow -Id $c.objectid -StateCode Activated -StatusCode Activated
-            }
```
[the remark this answers is not in the document]
Comment (pierluc-ssoucy) [reply azure-pipelines/scripts/update_connection_references.ps1:122]: we move to modern cmdlets for flows, started having issues with Flows and old cmdlets

File: src/Solution/Solution/Roles/Ci2 - Complaint Registrar.xml (hunk 5 above)
Comment (pierluc-ssoucy) [reply src/Solution/Solution/Roles/Ci2 - Complaint Registrar.xml:102]: yes

File: src/Solution/Solution/OptionSets/opc_sourcetype.xml [hunk 7]
```diff
@@ -24,7 +24,7 @@
     <option value="533950001" ExternalValue="">
       <labels>
         <label description="Workflow" languagecode="1033" />
-        <label description="" languagecode="1036" />
+        <label description="Workflow" languagecode="1036" />
```
[the remark this answers is not in the document]
Comment (pierluc-ssoucy) [reply src/Solution/Solution/OptionSets/opc_sourcetype.xml:27]: these are system optionsets for technical errors

File: src/Solution/Solution/OptionSets/opc_sourcetype.xml [hunk 8]
```diff
@@ -34,7 +34,7 @@
     <option value="533950002" ExternalValue="">
       <labels>
         <label description="Flow" languagecode="1033" />
-        <label description="" languagecode="1036" />
+        <label description="Flow" languagecode="1036" />
```
[the remark this answers is not in the document]
Comment (pierluc-ssoucy) [reply src/Solution/Solution/OptionSets/opc_sourcetype.xml:37]: these are system optionsets for technical errors

File: src/Solution/Solution/OptionSets/opc_sourcetype.xml [hunk 9]
```diff
@@ -14,7 +14,7 @@
     <option value="533950000" ExternalValue="">
       <labels>
         <label description="Plugin" languagecode="1033" />
-        <label description="" languagecode="1036" />
+        <label description="Plugin" languagecode="1036" />
```
[the remark this answers is not in the document]
Comment (pierluc-ssoucy) [reply src/Solution/Solution/OptionSets/opc_sourcetype.xml:17]: these are system optionsets for technical errors

**Pull request merged**